bundles/org.openhab.ui/web/src/assets/definitions/blockly/blocks-scripts.js

```diff
@@ -324,4 +325,30 @@ export default function defineOHBlocks_Scripts (f7, transformationServices) {
     const code = block.getFieldValue('inlineScript') + '\n'
     return code
   }
+
+  Blockly.Blocks['oh_rule_enable'] = {
+    init: function () {
+      this.appendValueInput('ruleUID')
+        .setCheck('String')
+        .appendField('Set Rule ')
+      this.appendValueInput('enable')
+        .appendField('as ')
```

Comment:
Not sure why the extra space is needed at the end - and in other places - since it seems to be perfectly fine without (with equal padding on both sides), but as it is apparently widespread in the code and the de-facto convention then okay (maybe I'm missing something!)